python/adapters/postgres_storage_adapter.py

```diff
@@ -53,78 +43,57 @@ def _commit(self):
     def _rollback(self):
         self.conn.rollback()
 
-    def convert_payment(self, source, destination, amount, memo, tx_fee, tx_charged_fee, op_index, tx_status, op_status,
-                        tx_hash, timestamp):
-        payment = dict.fromkeys(self.payments_output_schema())
-        payment['source'] = source
-        payment['destination'] = destination
-        payment['amount'] = amount
-        payment['memo_text'] = memo
-        payment['fee'] = tx_fee
-        payment['fee_charged'] = tx_charged_fee
-        payment['operation_index'] = op_index
-        payment['tx_status'] = tx_status
-        payment['op_status'] = op_status
-        payment['hash'] = tx_hash
-        payment['time'] = datetime.utcfromtimestamp(timestamp)
-
-        return payment
-
-    def convert_creation(self, source, destination, balance, memo, tx_fee, tx_charged_fee, op_index, tx_status,
-                         op_status, tx_hash, timestamp):
-        creation = dict.fromkeys(self.creations_output_schema())
-        creation['source'] = source
-        creation['destination'] = destination
-        creation['starting_balance'] = balance
-        creation['memo_text'] = memo
-        creation['fee'] = tx_fee
-        creation['fee_charged'] = tx_charged_fee
-        creation['operation_index'] = op_index
-        creation['tx_status'] = tx_status
-        creation['op_status'] = op_status
-        creation['hash'] = tx_hash
-        creation['time'] = datetime.utcfromtimestamp(timestamp)
-
-        return creation
+    def convert_operation(self, source, destination, amount, tx_order, tx_memo, tx_account, tx_account_sequence,
+                          tx_fee, tx_charged_fee, tx_status, tx_hash, op_order, op_status, op_type, timestamp,
+                          is_signed_by_app, ledger_file_name, ledger_sequence):
+
+        operation = dict.fromkeys(self.operation_output_schema())
+        operation['source'] = source
+        operation['destination'] = destination
+        operation['amount'] = amount
+        operation['tx_order'] = tx_order
+        operation['tx_memo'] = tx_memo
+        operation['tx_account'] = tx_account
+        operation['tx_account_sequence'] = tx_account_sequence
+        operation['tx_fee'] = tx_fee
+        operation['tx_charged_fee'] = tx_charged_fee
+        operation['tx_status'] = tx_status
+        operation['tx_hash'] = tx_hash
+        operation['op_order'] = op_order
+        operation['op_status'] = op_status
+        operation['op_type'] = op_type
+        operation['timestamp'] = datetime.utcfromtimestamp(timestamp)
+        operation['is_signed_by_app'] = is_signed_by_app
+        operation['ledger_file_name'] = ledger_file_name
+        operation['ledger_sequence'] = ledger_sequence
+
```

Comment:
Would this work for you?

```python
params = list(inspect.signature(a).parameters.keys())
loc = locals()
operation = {var: loc[var] for var in params}
operation['timestamp'] = datetime.utcfromtimestamp(timestamp)
```

Smaller and more flexible if you want to add more variables